models/project/github.go

```diff
@@ -0,0 +1,166 @@
+package project
+
+import (
+	"../../config"
+	"../../modules/hub"
+	"../../modules/logging"
+	"../db"
+	"../list"
+	"../task"
+	"database/sql"
+	"errors"
+
+	"github.com/google/go-github/github"
+)
+
+func (u *ProjectStruct) LoadFromGithub(issues []github.Issue) error {
+	for _, issue := range issues {
+		targetTask, _ := task.FindByIssueNumber(u.ID, *issue.Number)
+
+		err := u.taskApplyLabel(targetTask, &issue)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func IssuesEvent(repositoryID int64, body github.IssuesEvent) error {
```

Comment:
__[golint]__
* [golint] exported function IssuesEvent should have comment or be unexported